src/app-layout/__tests__/runtime-drawers.test.tsx

```diff
@@ -26,10 +28,10 @@ jest.mock('@cloudscape-design/component-toolkit', () => ({
 }));
 
 async function renderComponent(jsx: React.ReactElement) {
-  const { container, rerender } = render(jsx);
+  const { container, rerender, debug } = render(jsx);
```

Comment:
Need to remove this debug code.

--

P.S. pro tip for the future, this value is available as global export. This way it is easier to add and remove

```js
import { screen } from '@testing-library/react';

screen.debug();
```

Reply:
updated.